docs/dev/test-configuration-options.md

```diff
@@ -14,7 +14,7 @@ export const Highlight = ({children, color}) => ( <span style={{
 
 See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the code for setting the capabilities to execute a test.
 
-Depending on which environment you are running tests in, different options, also known as **Capabilities** (or options classes), should be set. There are different sets of capabilities for different environments, which can be combined. These configurations are added to the Capabilities or Options classes. Some of these setting are required for a test to run in a given environment, while some are optional. Certain capabilities
+Depending on which environment you are running tests in, different options, also known as **Capabilities**, should be set. There are different sets of capabilities for different environments, which can be combined. These configurations are added to the [Capabilities](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_Capabilities.html) or [Options](https://www.selenium.dev/documentation/en/driver_idiosyncrasies/driver_specific_capabilities/) classes. Some of these setting are required for a test to run in a given environment, while some are optional.
```

Comment:
I think it might feel more direct if you eliminate the passive voice? So, for example, 

```suggestion
Specify your test options, also known as **Capabilities**, according to your intended test environment, by adding them to the [Capabilities](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_Capabilities.html) or [Options](https://www.selenium.dev/documentation/en/driver_idiosyncrasies/driver_specific_capabilities/) classes. Whether certain capabilities are required or optional depends on the environment and the inter-dependency of the capabilities themselves.
```

Reply:
Rre-wrote this already